apis/rds/v1alpha1/generator-config.yaml

```diff
@@ -0,0 +1,24 @@
+ignore:
+  field_paths:
+    - DescribeDBClustersInput.DBClusterIdentifier
+    - CreateDBClusterInput.DBClusterIdentifier
+    - ModifyDBClusterInput.DBClusterIdentifier
+    - DeleteDBClusterInput.DBClusterIdentifier
+    - CreateDBClusterInput.DomainIAMRoleName
+    - ModifyDBClusterInput.DomainIAMRoleName
+    - CreateDBClusterInput.KmsKeyId
```

Comment:
```suggestion
```

You don't need to ignore referenced fields because we populate them with resolver result, hence it's fine for the code to use them. In custom struct, you need to add only the ref and selector fields.

Reply:
So I noticed the auto-generated code output `KMSKeyID` but in the aws sdk structs its  `KmsKeyId`. I am guessing  conversion takes care of this?

Reply:
So I noticed the auto-generated code output KMSKeyID but in the aws sdk structs its KmsKeyId. I am guessing conversion takes care of this?